client/my-sites/migrate/section-migrate.jsx

```diff
@@ -109,6 +109,9 @@ export class SectionMigrate extends Component {
 					...data.themes.filter( ( theme ) => ! theme.active ),
 				];
 				this.setState( { sourceSiteThemes } );
+			} )
+			.catch( ( err ) => {
+				/* suppress this error for now */
```

Comment:
As far as I can see, this function never works and must be replaced with 
![CleanShot 2023-09-20 at 17 13 32](https://github.com/Automattic/wp-calypso/assets/533/58889c75-39c2-4fb5-872c-140627dcee92)

I'll do that in a separate PR.